code/modules/organs/subtypes/standard_add.dm

```diff
@@ -0,0 +1,56 @@
+//For custom heads with custom parts since the base code is restricted to a single icon file.
+
+/obj/item/organ/external/head/vr/get_icon()
```

Comment:
I get the feeling that everything /head/vr does should probably just be a functionality change for /head.